config/install/oe_authentication.settings.yml

```diff
@@ -1,3 +1,4 @@
+langcode: en
```

Comment:
This fix makes sense.
I do find similar changes in core, and it aligns perfectly with the change record.